src/main/java/org/ibp/api/rest/germplasm/GermplasmResource.java

```diff
@@ -328,6 +328,17 @@ public long getFilteredCount() {
 		return new ResponseEntity<>(this.germplasmService.getGermplasmProgenitorDetails(gid), HttpStatus.OK);
 	}
 
+	@ApiOperation(value = "Get Germplasm used in lists other than specified.")
+	@PreAuthorize("hasAnyAuthority('ADMIN', 'CROP_MANAGEMENT', 'GERMPLASM', 'MANAGE_GERMPLASM')")
+	@RequestMapping(value = "/crops/{cropName}/germplasm/lists", method = RequestMethod.GET)
+	@ResponseBody
+	public ResponseEntity<Set<Integer>> getGermplasmPresentInOtherLists(@PathVariable final String cropName,
```

Comment:
We don't a new method to check (count or get) the GIDs present in lists. Let's avoid doing API methods like this and simply include in the warning message that deletion can take a long time if they are part of any germplasm lists - from which they will also be deleted.
Please remove all code related to this, also in MW and WB.